dpe/src/dpe_instance.rs

```diff
@@ -39,7 +39,7 @@ pub struct DpeEnv<'a, T: DpeTypes + 'a> {
 }
 
 #[repr(C, align(4))]
-#[derive(AsBytes, FromBytes, Zeroize)]
+#[derive(AsBytes, FromBytes, ZeroizeOnDrop)]
```

Comment:
I want to be careful with DpeInstance since it in DCCM. Really it should never get dropped since we only ever have borrows of it.

Reply:
Hmm yeah, I guess we don't really need this PR then. Abandoning.